verify/verify_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package verify
+
+import (
+	"testing"
+)
+
+func TestVerifyProvenanceFile(t *testing.T) {
+	path := "../testdata/provenances/15dc16c42a4ac9ed77f337a4a3065a63e444c29c18c8cf69d6a6b4ae678dca5c.json"
+
+	if err := Verify(path, ""); err != nil {
+		t.Fatalf("couldn't verify the provenance file: %v", err)
+	}
```

Comment:
It would be nice if this test could receive and check the actual hash of the build being perform. At the present it implicitly relies on `Verify` checking the hash.

Reply:
This test is checking the functionality of `Verify`. I think the scenario you have described can be another, finer-grained test to be added to `common`, as part of #13.